### PR TITLE
Fixes #15642: Make rudder-setup work on aix

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15642

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
PR updated with a new commit

File: scripts/rudder-setup/setup-agent.sh [hunk 1]
```diff
@@ -47,19 +53,19 @@ setup_agent() {
...
-    echo "${SERVER}" > /var/rudder/cfengine-community/policy_server.dat
-    if is_version_valid "${RUDDER_VERSION}" "[3.0 *]"
-    then
-      rudder agent run
-    else
-      cf-agent -K -D force_inventory -b doInventory
-    fi
+    rudder agent inventory
+  else
+    cf-agent -K -D force_inventory -b doInventory
```
Comment (amousset) [inline scripts/rudder-setup/setup-agent.sh]: We could probably drop support for pre-3.0

Comment (peckpeck) [reply scripts/rudder-setup/setup-agent.sh]: let's do it in a different PR to see how much code it removes

**Pull request merged**